CGPs/0013.md

```diff
@@ -34,18 +34,14 @@ This Celo Core Contracts Upgrade proposal is special since it operationalizes th
 16. UsingPrecompilesProxy `setImplementation`
 17. ValidatorsProxy `setImplementation`
 
-Additionally, to avoid future updated bytecode of Celo Core Contracts when underlying libraries are changed, some libraries are being proxied and linked via the proxy. That necessitates the following transactions:
-
-1. Registry `setAddressFor` Signatures library
-2. Registry `setAddressFor` AddressLinkedList library
-3. Registry `setAddressFor` AddressSortedLinkedList library
-4. Registry `setAddressFor` AddressSortedLinkedListWithMedian library
-5. Registry `setAddressFor` Proposals library
-6. Registry `setAddressFor` IntegerSortedLinkedList library
-
 Regular features that were being added or bugs that were fixed are described in the [Github release](https://github.com/celo-org/celo-monorepo/releases/tag/celo-contracts-v1.rc1). Since the major version of these Celo Core Contracts changed due to the version change, no separate transactions will reflect the "regular" changes. One exception to that is the newly added `DowntimeSlasher`:
 
 1. Registry `setAddressFor` DowntimeSlasher
+2. DowntimeSlasherProxy `setAndInitializeImplementation`:
```

Comment:
is this relevant to not proxying libraries?

Reply:
Not quite, its part of us moving initialization into the proposal itself